lib/structuraid_core/engineering/analysis/footing/centric_combined_two_columns.rb

```diff
@@ -0,0 +1,85 @@
+module StructuraidCore
+  module Engineering
+    module Analysis
+      module Footing
+        class CentricCombinedTwoColumns < Base
+          include Utils::Data
+          include Utils::Centroid
+          include Utils::ShearMoment
+
+          def initialize(footing:, loads_from_columns:, section_direction:)
+            if ORTHOGONALITIES.none?(section_direction)
+              raise Engineering::Analysis::SectionDirectionError.new(section_direction, ORTHOGONALITIES)
+            end
+
+            @footing = footing
+            @loads_from_columns = loads_from_columns
+            @section_direction = section_direction
+          end
+
+          def build_geometry(coordinates_system)
+            @coordinates_system = coordinates_system
```

Comment:
Do we need to have a `coordinates_system` attribute on this abstraction besides the `coordinate_system` of the footing?

Reply:
No, I agree.
I was passing the footing.coordinatess_system as an argument, but it doesn´t make sense if we have the footing as an attribute of the CentricCombinedTwoColumns.

So from now, there is no `coordinates_system`attribute; we access the coordinates system via `footing`

<img width="938" alt="image" src="https://user-images.githubusercontent.com/26731448/228001289-d788868b-0fe2-432a-ae69-f8f727916ac4.png">

Reply:
ok! cool. Another small refactor could be to have a separate method `coordinates_system` that returns the footing's coordinates_system. That way you don't need to create a local variable every time you need to use it

Reply:
I like it, done!